etna/records/fields.py

```diff
@@ -20,6 +22,9 @@ class LazyRecord(SimpleLazyObject):
     ``id``, or requesting a string representation of it.
     """
 
+    # Broken image upload.
+    # widget = RecordChooser
```

Comment:
Why's this broken and need commenting out?

Reply:
I think it was broken before because of my issue with URL registering breaking the admin URLs reversing.